src/CloudSerial.cpp

```diff
@@ -14,6 +14,7 @@
    software without disclosing the source code of your own applications. To purchase
    a commercial license, send an email to license@arduino.cc.
 */
+#if defined(ARDUINO_SAMD_MKRGSM1400) || defined(ARDUINO_SAMD_MKRWIFI1010) ||  defined(ARDUINO_SAMD_MKR1000) || defined(ARDUINO_SAMD_NANO_33_IOT) || defined(ARDUINO_ESP8266_ESP12) || defined(ARDUINO_ARCH_ESP32) || defined(ESP8266)
```

Comment:
@fabik111 Why is `ARDUINO_SAMD_MKRNB1500` not here?

Wouldn't it be better if we use an `ifndef` instead?
```suggestion
#ifndef HAS_LORA
```
Like this, it wouldn't be necessary to edit this line every time we add a new board (and it's less likely that we forget to add some)

Reply:
Fixed, pay attention to the future boards that will not support cloudSerial